coalib/bearlib/languages/definitions/flow.py

```diff
@@ -0,0 +1,12 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class flow:
```

Comment:
I would suggest capitalizing the initial letter of the class name. (ref: https://www.python.org/dev/peps/pep-0008/#class-names)